org.ekstep.mathtext-1.0/editor/mathtext.js

```diff
@@ -754,13 +758,35 @@ angular.module('org.ekstep.mathtext', [])
       return divElement.textContent || divElement.innerText;
     }
 
+     $timeout( function() {
```

Comment:
Use the existing $timeout function instead of creating one more timeout function.